extensions/reactive-datasource/deployment/src/main/java/io/quarkus/reactive/datasource/deployment/ReactiveDataSourceProcessor.java

```diff
@@ -0,0 +1,5 @@
+package io.quarkus.reactive.datasource.deployment;
+
+public class ReactiveDataSourceProcessor {
```

Comment:
Is this empty class needed for anything?

Reply:
See previous comment.

Reply:
See previous answer :P